Composer/packages/ui-plugins/virtual-assistant-creation/src/shared/util.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import axios from 'axios';
+import { ShellApi } from '@bfc/shared';
+
+import { AvailablePersonalities } from '../models/creationOptions';
+
+// Utility constants
+const GitHubRepoApiRootUrl =
+  'https://api.github.com/repos/microsoft/BotBuilder-PersonalityChat/contents/CSharp/Datasets/qnaFormat/';
+
+// GitHub API Utility Functions
+export const updatePersonalityQnaFile = async (shellApi: ShellApi, selectedPersonality: AvailablePersonalities) => {
+  const personalityFileName = `qna_chitchat_${selectedPersonality}.qna`;
+  // TODO: handle different markets once creation supports multi-local
```

Comment:
If you want to check-in with a TODO, consider creating a GitHub issue and referencing it here.

Reply:
Changed the comment from a todo to a more descriptive comment